src/app/auth/store/auth.selectors.ts

```diff
@@ -0,0 +1,19 @@
+import { createSelector } from '@ngrx/store';
+
+import { AppState } from '../../store/app-state.model';
+import { AuthState } from './auth-state.model';
+
+export const getUser = createSelector(
+  (state: AppState) => state.authFeature,
+  (state: AuthState) => state.user
+);
+
+export const getIsAuth = createSelector(
+  (state: AppState) => state.authFeature,
+  (state: AuthState) => state.isAuth
+);
+
+export const getError = createSelector(
+  (state: AppState) => state.authFeature,
+  (state: AuthState) => state.error
+);
```

Comment:
use feature selector to avoid `(state: AppState) => state.authFeature` in every selector.